src/hooks/rest-api.tsx

```diff
@@ -13,7 +32,18 @@ const hooks = createAPIHooks<RestAPIEndpoints>({
   },
 });
 
-export const useRestQuery = hooks.useAPIQuery;
+export const useRestQuery: <
+  Route extends keyof Endpoints & string,
+  Data = Endpoints[Route]['Response'],
+>(
+  route: Route,
+  payload: RequestPayloadOf<Endpoints, Route>,
+  options?: RestrictedUseQueryOptions<
+    Endpoints[Route]['Response'],
+    unknown,
+    Data
+  >,
```

Comment:
Why did we need this explicit type? I think we didn't have it when we used the library on the FL member app.

Reply:
See https://lifeomic.slack.com/archives/C05KMKQNTAR/p1707948259517759 or the failed earlier builds. Happy to fix some other way but that is what was decided there.

Reply:
We are not building the FL member app as an SDK is probably the biggest difference.